### PR TITLE
added kaina week 5 homework

### PR DESCRIPTION
**Pull request by kaina1129**

### CODE REVIEW

File: week5/kaina/TAOBAO/index.css [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+.login-input-box {
+  display: flex;
+  position: relative;
+}
+
+.login-input-box input {
+  width: 250px;
+  height: 23px;
+  font-size: 14px;
+  padding: 9px 8px 8px 50px;
```
Comment (mxdi9i7) [inline week5/kaina/TAOBAO/index.css]: why is the top paddnig 9px?

File: week5/kaina/TAOBAO/index.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+}
+
+.help-footer-web a {
+  padding: 0px 10px 0px 5px;
+  border-right: 1px solid #485065;
+  margin-left: -2px;
+}
+
+.help-footer-about a {
+  margin: 0px 2px;
```
Comment (mxdi9i7) [inline week5/kaina/TAOBAO/index.css]: an inline element `<a>` does not have margin top or bottom, therefore you don't need to specify that the a tag has a 0 margin top and bottom. Come up with a more efficient way to style this a tag.

File: week5/kaina/TAOBAO/index.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+}
+
+.help-footer-license p {
+  color: #9c9c9c;
+  font-family: "Microsoft YaHei", tahoma, arial, "Hiragino Sans GB",
+    "\5b8b\4f53", sans-serif;
+  font-size: 12px;
+  display: inline;
+  padding: 0px 10px 0px 6px;
+  margin-left: -4px;
```
Comment (mxdi9i7) [inline week5/kaina/TAOBAO/index.css]: negative margin?

File: week5/kaina/TAOBAO/index.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Kaina</title>
```
Comment (mxdi9i7) [inline week5/kaina/TAOBAO/index.html]: title should be taobao's page title, not Kaina

File: week5/kaina/TAOBAO/index.html [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+          src="http://gtms04.alicdn.com/tps/i4/TB14nhUIVXXXXXAXpXXlu.FQpXX-300-124.png"
+          alt="taobao logo"
+        />
+        <p>账户登录</p>
+      </div>
+    </nav>
+    <div class="wrapper">
+      <div class="login-container">
+        <div class="login-box">
+          <img src="scancode.png" alt="" />
```
Comment (mxdi9i7) [inline week5/kaina/TAOBAO/index.html]: missing alt

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]
where is your Qzone and JS homework? I'm only seeing taobao

**Comment by mxdi9i7** [comment]
Your taobao quicklogin QR is not clickable.

**Comment by mxdi9i7** [comment]
Also, please remove all .DS_Store in all folder before you commit and push your code

File: week5/kaina/qz/index.css [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+  background-size: cover;
+  font-family: helvetica, arial, 微软雅黑, 华文黑体;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+* {
+  font-family: "microsoft Yahei", "微软雅黑", SimHei, "黑体", "Hiragino Sans GB";
```
Comment (mxdi9i7) [inline week5/kaina/qz/index.css]: Microsoft, cap the M

File: week5/kaina/qz/index.css [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+
+.logo-container {
+  width: 100%;
+  max-width: 880px;
+  position: absolute;
+  top: 95px;
+}
+
+.logo-container img {
+  width: 251px;
```
Comment (mxdi9i7) [inline week5/kaina/qz/index.css]: width need to be even numbers

File: week5/kaina/qz/index.css [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+}
+
+.quick-login-container p {
+  margin: 0px;
+  color: #666666;
+}
+
+.log-in-container a {
+  text-decoration: none;
+  color: #3481cf;
```
Comment (mxdi9i7) [inline week5/kaina/qz/index.css]: this code looks repeated from line 105/106
can you merge the two?

File: week5/kaina/qz/index.html [hunk 9]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>QQ空间-分享生活，留住感动</title>
+    <link
+      rel="Shortcut Icon"
+      href="//qzonestyle.gtimg.cn/aoi/img/logo/favicon.ico?max_age=31536000"
```
Comment (mxdi9i7) [inline week5/kaina/qz/index.html]: why does this icon url start with `//`?

File: week5/kaina/qz/index.html [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+        <img
+          src="https://qzonestyle.gtimg.cn/qzone_v6/proj_qzonelogin/img/qzone-login-logo-dark.32@2x.png"
+          alt="logo"
+        />
+        <div class="quick-login-container">
+          <h3>快速安全登录</h3>
+          <p>请使用<a href="#">QQ手机版</a>扫描二维码，</p>
+          <p>安全登录，防止盗号。</p>
+          <div class="form-container">
+            <img src="QRCODE.png" alt="" />
```
Comment (mxdi9i7) [inline week5/kaina/qz/index.html]: empty alt

File: week5/kaina/week5js/index.js [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+};
+
+// console.log(kaina.age);
+// console.log(kaina.pet, kaina.college.location);
+// document.write(kaina.favoriteFood, kaina.pet, kaina.age);
+
+const isString = kaina => {
+  if (typeof kaina == "string") {
+    return true;
+  } else {
```
Comment (mxdi9i7) [inline week5/kaina/week5js/index.js]: don't need the else here

File: week5/kaina/week5js/index.js [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+  }
+};
+// console.log(isString("future sphere"));
+// console.log(isString(35));
+// console.log(isString("will"));
+
+const isBlank = input => {
+  if (input.length == 0) {
+    return true;
+  } else {
```
Comment (mxdi9i7) [inline week5/kaina/week5js/index.js]: no need to have else

File: week5/kaina/week5js/index.js [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+    return space.slice(1);
+  } else if (space.substring(space.length - 1) == " ") {
+    return space.substring(0, space.length - 1);
+  }
+};
+// console.log(stripSpace(" Ronbin Hood from China"));
+// console.log(stripSpace("Ronbin Hood from China "));
+// console.log(stripSpace(" Ronbin Hood from China "));
+
+// const formatPhoneNumber = num => {
```
Comment (mxdi9i7) [inline week5/kaina/week5js/index.js]: where is format phone?

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]

File: week5/kaina/TAOBAO/index.css (hunk 1 above)
Comment (knazhang) [reply week5/kaina/TAOBAO/index.css]: updated

File: week5/kaina/TAOBAO/index.css (hunk 2 above)
Comment (knazhang) [reply week5/kaina/TAOBAO/index.css]: inline block

File: week5/kaina/TAOBAO/index.css (hunk 3 above)
Comment (knazhang) [reply week5/kaina/TAOBAO/index.css]: updated

File: week5/kaina/qz/index.css (hunk 6 above)
Comment (knazhang) [reply week5/kaina/qz/index.css]: 🉑️

File: week5/kaina/qz/index.html (hunk 9 above)
Comment (knazhang) [reply week5/kaina/qz/index.html]: ok update

File: week5/kaina/qz/index.html (hunk 10 above)
Comment (knazhang) [reply week5/kaina/qz/index.html]: OK

**Review by mxdi9i7 — APPROVED** [review]

**Pull request merged**